bin/deploy.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+if [[ -z $1 || -z $2 ]]; then
+    echo "USAGE: $0 [prod|dev] <app-dir>"
+    exit 1
+fi
+
+## Get CLI parameters.
+MODE=$1
+APP_DIR=$(cd $2; pwd)
+
+## Update the repo.
+cd $APP_DIR
+git fetch --quiet origin 2>&1
+
+## Get some information about git.
+HIGHEST_TAG=$(git tag | sort --version-sort | tail --lines 1)
```

Comment:
I would recommend passing `-l` to `git tag` to make it clear that we are listing even though listing is the default no flag behavior. We might also use the `--sort` flag on `git tag` instead of piping to `sort` if that proves more reliable and/org safer.